src/Vault.sol

```diff
@@ -746,13 +758,8 @@ contract Vault is EIP1967Admin, VaultAccessControl, IERC721Receiver {
 
     /// @notice Close a vault (internal)
     /// @param vaultId Id of the vault
-    /// @param owner Vault owner
     /// @param nftsRecipient Address to receive nft of the positions in the closed vault
-    function _closeVault(
-        uint256 vaultId,
-        address owner,
-        address nftsRecipient
-    ) internal {
+    function _closeVault(uint256 vaultId, address nftsRecipient) internal {
```

Comment:
I am not sure, why the nft isn't burned here?

Reply:
We don't have to burn it anywhere in the contract. A user can deposit collateral in the vault after it's closure